srcpkgs/musikcube/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'musikcube'
+pkgname=musikcube
+version=0.60.1
+revision=1
+build_style=cmake
+configure_args="-DNO_NCURSESW=1"
+makedepends="boost-devel libogg-devel libmicrohttpd-devel ffmpeg-devel
+ libressl-devel pulseaudio-devel lame-devel zlib-devel alsa-lib-devel
+ libvorbis-devel libcurl-devel ncurses-devel libev-devel taglib-devel"
+short_desc="Terminal-based music player in c++"
+maintainer="eater <=@eater.me>"
+license="BSD-3-Clause"
+homepage="https://musikcube.com/"
+distfiles="https://github.com/clangen/musikcube/archive/${version}.tar.gz"
+checksum=7b6661673952a0a27310053ad19defea313a70f4c70aecc91e4a5d1a71bde13c
+
+if [[ "${XBPS_TARGET_MACHINE}" == armv[56]* ]]; then
```

Comment:
it is more common to use a `case esac` structure for dealing with these

Reply:
fixed :)